src/name_service/delivery/utils.clj

```diff
@@ -0,0 +1,40 @@
+(ns name-service.delivery.utils
+  (:require [cheshire.core :as json]
+            [io.pedestal.http :as bootstrap]
+            [io.pedestal.http.body-params :as body-params]
+            [name-service.core.action :as action]
+            [name-service.delivery.http :as http]))
+
+(defn merge-with-payload->json [coll payload]
+  (->>
+    coll
+    (merge payload)
+    (json/encode)))
+
+(defn inject-interceptors [service-map]
+  (-> service-map
+    bootstrap/default-interceptors
+    (update ::bootstrap/interceptors conj (body-params/body-params))
+    (update ::bootstrap/interceptors conj bootstrap/html-body)))
+
+(defn keymap->arguments [keymap]
+  (->> keymap
+    (clojure.walk/stringify-keys)
+    (mapcat identity)
+    (vec)))
+
+(defn action->response
```

Comment:
Make sure you don't duplicate code of Mateusz.
(This should be sorted out when merging, but depends who will be first)

Reply:
Yes, I am aware of that.